dev/kyuubi-tpcds/src/main/scala/org/apache/kyuubi/tpcds/benchmark/RunBenchmark.scala

```diff
@@ -66,11 +63,16 @@ object RunBenchmark {
       opt[String]('r', "results-dir")
         .action((x, c) => c.copy(resultsDir = x))
         .text("dir to store benchmark results, e.g. hdfs://hdfs-nn:9870/pref")
-      opt[String]('q', "queries")
+      opt[String]('I', "include")
```

Comment:
the short opt is confusing, let's keep the long one only

Reply:
updated.